crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -91,7 +88,18 @@ impl RocksDb {
                         }
                         Ok(db)
                     }
-                    err => err,
+                    _ => {
+                        DB::repair(&opts, &path)
```

Comment:
Maybe we should add some logging here so we know if the database is going into repair mode?